utils/search/search_general.py

```diff
@@ -85,41 +88,43 @@ def search_process_filter(filter_query):
     return filter_query
 
 
-def split_filter_query(filter_query, cluster_id):
-    """Parses and pre-process search filter parameters and returns the filters' information.
+def split_filter_query(filter_query, parsed_filters, cluster_id):
+    """Pre-process parsed search filter parameters and returns the filters' information.
 
     This function is used in the search template to display the filter and the link when removing them.
     The clustering facet filter is in a separate query parameter. It facilitates to distinguish them from
```

Comment:
`cluster_id `is indeed not part of the filters. It is passed using an additional url query parameter.
I think having it separated from the other used to make sense when we did not have a good parser for the filters. Now it is true that the comment does not really make sense, because it is not really facilitating anything. Well, maybe it is still good to have the clustering filter very explicitly separated, because it's a different kind of filter that is not directly passed to Solr as such. I will update the docstring.